src/data/Company.java

```diff
@@ -0,0 +1,70 @@
+package data;
+
+import utils.FileOperations;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Company {
+    private final String name;
+    private double minimumSalary = Double.MAX_VALUE;
+    private double maximumSalary = Double.MIN_VALUE;
+    private double averageSalary = 0;
+    private double employeeCounter = 0;
+    private ArrayList<Department> departments;
+
+    public Company(String name) {
+        this.name = name;
+    }
+
+    public void getCompanyStatistics(String fileName) throws IOException {
```

Comment:
skoro ta metoda ustawia większość pól w tej klasie to powinna być częscią konstruktora. Rozdzielanie konstrukcji obiektu na dwie metody(bo musisz uzyc konstruktora a pozniej tą metode zeby tak naprawde zainicjalizowac obiekt) jest niepoprawne

Reply:
Ok wrzuciłem wywołanie do konstruktora, szczerze to zrobiłem to tak, bo sam byłem ciekaw co powiesz :) Nie umiałem sam sobie odpowiedzieć czy to poprawnie czy nie.